src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -144,7 +146,17 @@ const PermissionIdentities: FC = () => {
           "aria-label": "Auth method",
         },
         {
-          content: identity.type,
+          content: (
+            <ResourceLink
```

Comment:
We can use the `ResourceLabel` component here since we are not linking each identity to another page.

Reply:
nothing changed here

Reply:
This is still using `ResourceLink` and not `ResourceLabel`?